nixos/modules/system/activation/activation-script.nix

```diff
@@ -233,10 +233,11 @@ in
     system.activationScripts.stdio = ""; # obsolete
     system.activationScripts.var = ""; # obsolete
 
-    systemd.tmpfiles.rules = [
+    systemd.tmpfiles.rules = lib.optionals config.nix.enable [
```

Comment:
We should swap the none conditional files to be in the front

Reply:
I reordered them to have these last